src/gtk/clipbrd.cpp

```diff
@@ -663,10 +674,38 @@ bool wxClipboard::AddData( wxDataObject *data )
     // for explanation
     AddSupportedTarget(g_timestampAtom);
 
+#ifdef __WXGTK3__
+    static const bool isWayland =
```

Comment:
The `IsWayland()` result is already cached, so there is no need to make this static.